extra/modules/README.md

```diff
@@ -0,0 +1 @@
+# Prebid Server modules
```

Comment:
This doesn't give this section enough credit ☺️

The blocking module is amazing. Adding docs on how to enable and configure it would be highly appreciated ❤️